routes/transcription.py

```
@@ -165,6 +172,9 @@ async def add_to_queue(
             "status": "queued",
             "message": "Transcription source has been added to the queue.",
         }
+    except ValueError as e:
+        logger.error(f"Validation error: {e}")
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)